cinema/models.py

```diff
@@ -27,8 +27,12 @@ class Actor(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
 
+    @property
+    def full_name(self):
+        return f"{self.first_name} {self.last_name}"
+
     def __str__(self):
```

Comment:
what about type annotations?

Reply:
okaayyyyy :) i forgot)))

Reply:
unfortunately, almost like everyone else. That's why it's ok 😸